configs/11.0/packages/binutils/sources

```diff
@@ -49,11 +49,19 @@ atsrc_get_patches ()
 		https://sourceware.org/ml/binutils/2017-02/msg00269.html \
 		fix-incorrect-indirection.patch 14 \
 		80e3c4062604f28f186289db0167b592 || return ${?}
+
+	at_get_patch_and_trim \
+		https://sourceware.org/ml/binutils/2017-03/msg00173.html \
+		fix-urcu-rpath-error.patch 22 \
```

Comment:
@tuliom Is not really a rpath. I give this name before based on this line of the reported error: `try using -rpath or -rpath-link`. The problem happens on dynamic linkage of libs on binutils due a wrong search of libs.